### DISCUSSION TITLE
PFX file

### DISCUSSION THREAD

**Discussion by pagatone** [openssl/openssl#25654] [Q&A] [answered]
I am using OpenSSL 3 stable (3.3.2) and i need to know how to convert p12 file to old pfx format. Do i just have to do only  
openssl pkcs12 -in FILE.p12 -out FILE.pfx  
or is there some special option like -legacy or something. I am asking because new OpenSSL working only with 3 formats, DER, PEM and p12.

**Comment by mattcaswell** [comment, marked as answer]
To all intents and purposes a .p12 file and a .pfx file are the same thing. They both conform to the PKCS#12 format. You can just rename the file.

You can read more about the history of why we have two file extensions for the same thing here:
https://openssl-users.openssl.narkive.com/DjQR3M98/p12-vs-pfx
